.github/workflows/deploy_devnet.yml

```diff
@@ -0,0 +1,230 @@
+            --output text \
+            --query "Command.CommandId")
+          echo "command_id=$COMMAND_ID" >> $GITHUB_OUTPUT
+
+      - name: Check stop services command status and retrieve output
+        run: |
+          sleep 5
+          STATUS=$(aws ssm list-command-invocations \
+              --command-id ${{ steps.stop_services.outputs.command_id }} \
+              --details \
+              --query "CommandInvocations[0].Status" \
+              --output text)
+          echo "Command Status: $STATUS"
+          OUTPUT=$(aws ssm list-command-invocations \
+              --command-id ${{ steps.stop_services.outputs.command_id }} \
+              --details \
+              --query "CommandInvocations[0].CommandPlugins[0].Output" \
+              --output text)
+          echo "Command Output: $OUTPUT"
+          if [ "$STATUS" != "Success" ]; then
+              echo "Command failed with status: $STATUS"
+              exit 1
+          fi
+      
+      - name: Install prerequisites
+        id: install_prereq
+        run: |
+          COMMAND_ID=$(aws ssm send-command \
+            --instance-ids ${{ secrets.DEVNET_INSTANCE_ID }} \
+            --document-name "AWS-RunShellScript" \
+            --parameters '{"commands":[
+                  "sudo apt-get update -y",
+                  "sudo apt-get install -y git gcc openssl libbz2-dev libffi7 make cargo",
+                  "sudo apt install -y build-essential",
+                  "curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y",
+                  "source $HOME/.cargo/env",
+                  "rustup update",
+                  "sudo apt-get install -y clang cmake",
+                  "sudo rm -rf /opt/miden/miden-node/genesis.dat",
+                  "sudo rm -rf /opt/miden/miden-node/miden-store.sqlite3",
+                  "sudo rm -rf /opt/miden/miden-node/miden-store.sqlite3-*",
+                  "sudo rm -rf /opt/miden/miden-node/accounts",
+                  "sudo rm -rf /opt/miden/miden-node/blocks"
+                  ]}' \
+            --output text \
+            --query "Command.CommandId")
+          echo "command_id=$COMMAND_ID" >> $GITHUB_OUTPUT
+  
+      - name: Poll command status and retrieve live logs from install prereqs
+        run: |
+          while true; do
+            STATUS=$(aws ssm list-command-invocations \
+                --command-id ${{ steps.install_prereq.outputs.command_id }} \
+                --details \
+                --query "CommandInvocations[0].Status" \
+                --output text)
+            echo "Command Status: $STATUS"
+            
+            OUTPUT=$(aws ssm list-command-invocations \
+                --command-id ${{ steps.install_prereq.outputs.command_id }} \
+                --details \
+                --query "CommandInvocations[0].CommandPlugins[0].Output" \
+                --output text)
+            echo "Command Output: $OUTPUT"
```

Comment:
When called in a loop, does this only contain the new output, or is it the full output each time?

As in, if I call it twice it will print the same thing twice or only the new output is appended? I assume the former.